tools/pylint-extensions/azure-pylint-guidelines-checker/tests/test_files/do_not_hardcode_connection_verify_acceptable.py

```diff
@@ -0,0 +1,48 @@
+class InstanceVariableError:
+    def __init__(self):
+        self.connection_verify = None
+        self.self = self
+
+
+class VariableError:
+    connection_verify = None
```

Comment:
can we remove these they are not related overload rule :)

Reply:
For our final report we tried to put all of our checkers in one branch so that code coverage reports were easier, this didn't work, it changed our pull requests - because there has been so many changes to the files since I can't revert the pull - I'm going to create a V2 branch with the below removed on a new pull request.